package/MDAnalysis/analysis/rms.py

```diff
@@ -358,8 +358,19 @@ class RMSD(AnalysisBase):
     .. versionchanged:: 2.0.0
        :attr:`rmsd` results are now stored in a
        :class:`MDAnalysis.analysis.base.Results` instance.
-
+    .. versionchanged:: 2.8.0
+       introduced a :meth:`get_supported_backends` allowing for execution on with
+       ``multiprocessing`` and ``dask`` backends.
     """
+
+    @classmethod
+    def get_supported_backends(cls):
+        return ('serial', 'multiprocessing', 'dask',)
+
+    @classmethod
+    def _is_parallelizable(cls):
+        return True
```

Comment:
The current documentation https://mdanalysis--4162.org.readthedocs.build/en/4162/documentation_pages/analysis/parallelization.html#adding-parallelization-to-your-own-analysis-class states that the method to change is called `is_parallelizable()` (no leading underscore):
```
    @classmethod
    def is_parallelizable(self):
        return True
```

Docs and code need to be consistent.

(In any case, as said in other comments, I'd prefer an attribute `parallelizable` to be consistent with the transformations.)

Reply:
Fixed; also see discussion above.